orangecontrib/spectroscopy/widgets/owspectra.py

```diff
@@ -694,12 +698,12 @@ def __init__(self, parent: OWWidget, select=SELECTNONE):
         self.clear_data()
         self.subset = None  # current subset input, an array of indices
         self.subset_indices = None  # boolean index array with indices in self.data
-
+        self.start_point = None  # automatic position for adding single lines in peak_apply
```

Comment:
`start_point` should not be a module attribute, as it can be quickly computed when needed (is totally data dependent).